test/acceptance/pages.py

```diff
@@ -557,11 +570,24 @@ def learner_response(self):
             css=self._bounded_selector(".staff-info__student__response .ui-toggle-visibility__content")
         ).text[0]
 
-    def submit_assessment(self):
+    def staff_assess(self, options_selected, continue_after=False):
+        for criterion_num, option_num in enumerate(options_selected):
+            sel = "#staff__assessment__rubric__question--{criterion_num}__{option_num}".format(
+                assessment_type="staff",
+                criterion_num=criterion_num,
+                option_num=option_num
+            )
+            self.q(css=self._bounded_selector(sel)).first.click()
+        self.submit_assessment(continue_after)
+
+    def submit_assessment(self, continue_after=False):
         """
         Submit a staff assessment of the problem.
         """
-        self.submit(button_css=".wrapper--staff-assessment .action--submit")
+        filter_text = "Submit Assessment"
+        if continue_after:
+            filter_text += " and Grade Another Learner"
+        self.q(css=self._bounded_selector("button.action--submit")).filter(text=filter_text).first.click()
 
     def cancel_submission(self):
```

Comment:
Can you add coverage for the "ungraded" and "checked out" numbers? That might be tricky because there can be extra submissions in the system, so we might just need to get the numbers from the start of the test and increment/decrement from that point on.